docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+name: "wisee-backend"
+
+services:
+    wisee-DB:
+        image: postgres:latest
+        ports:
+            - "7856:5432"
+        environment:
+            POSTGRES_USER: postgres
+            POSTGRES_PASSWORD: postgres
+            POSTGRES_DB: wisee
+    wisee-DB-test:
```

Comment:
why the test db is here, and any plan to drop the database, id yes at what time.